dask/array/rechunk.py

```diff
@@ -160,10 +160,13 @@ def old_to_new(old_chunks, new_chunks):
     Handles missing values, as long as the dimension with the missing chunk values
     is unchanged.
 
+    Notes
+    -----
     This function expects that the arguments have been pre-processed by
     :func:`dask.array.core.normalize_chunks`. In particular any ``nan`` values should
     have been replaced (and are so by :func:`dask.array.core.normalize_chunks`)
-    by the canonical ``np.nan``.
+    by the canonical ``np.nan``. It also expects that the arguments have been validated
+    with `_validate_rechunk` and rechunking is thus possible.
```

Comment:
Nit: we want double backticks around ``_validate_rechunk`` for sphinx

Reply:
Is there a good way to lint this? 🤔

Reply:
Not that I've run into (though I've also not specifically looked for one). Though I'd definitely use it if one existed!